observations/obs_converters/AIRS/airs_obs_mod.f90

```diff
@@ -72,7 +71,8 @@ module airs_obs_mod
 
 contains
 
-!-------------------------------------------------
+!-------------------------------------------------------------------------------
+!>
```

Comment:
No functional changes in this file ... just code cleanup and removing unused variables.